multisense_ros/msg/AprilTagDetectionArray.msg

```diff
@@ -0,0 +1,16 @@
+std_msgs/Header header
+
+# The frame ID of the image that the apriltags were detected on
+int64 frameId
```

Comment:
I am torn on this. I think it makes sense to publish, but this might be the only spot where we actually explicitly publish the frameId. The sensor_msgs::Image does not have a field for this

Reply:
Removed, as I already use this frameID as the seq value in the header.